lib/woql.js

```diff
@@ -92,17 +92,62 @@ WOQL.and = function(...subqueries) {
 
 
 /**
- * Read a node identified by an IRI as a JSON-LD document
+ * [Instead use read_document] Read a node identified by an IRI as a JSON-LD document
  * @param {string} IRI -  The document id  or a variable
  * @param {string} output - variable name
  * @param {object} formatObj 
  * @return {object} WOQLQuery
+ * @deprecated 
  */
 
 WOQL.read_object = function(IRI, output, formatObj) {
     return new WOQLQuery().read_object(IRI, output, formatObj)
 }
 
+/**
+ * Read a node identified by an IRI as a JSON-LD document
+ * @param {string} IRI -  The document id  or a variable
+ * @param {string} output - variable name
+ * @param {object} formatObj 
+ * @return {object} WOQLQuery
+ */
+
+ WOQL.read_document = function(IRI, output, formatObj) {
+    return new WOQLQuery().read_document(IRI, output, formatObj)
+}
+
+/**
+ * Insert a document in the graph.
+ * @param {object} docjson -  The document to insert. Must either have an '@id' or have a class specified key.
+ * @param {string} [IRI] - An optional returned identifier specifying the documentation location.
```

Comment:
`An optional identifier specifying the document location.` I realize that this is wrong in the `woql.json`.